Telemetry/Telemetry.json

```diff
@@ -58,7 +58,13 @@
                 ]
             },
             "result": {
-                "$ref": "#/common/result"        
+                "$ref": "#/common/result"
+            }
+        },
+        "uploadReport": {
+            "summary": "Triggers processing and uploading of telemetry report for legacy Xconf based configuration",
+            "result": {
+                "$ref": "#/common/result"
```

Comment:
please do not use XRE Service manager object "#/common/result"

in Thunder there are error codes

e.g.

https://github.com/rdkcentral/rdkservices/blob/609ca41faf9b872b4deddf7bec23c8fde8aec4b4/AVInput/AVInput.json#L171